code/__defines/_planes+layers.dm

```diff
@@ -136,16 +136,16 @@ What is the naming convention for planes or layers?
 
 //HUD layer defines
 
-#define FULLSCREEN_PLANE 18
+#define FULLSCREEN_PLANE 32000
 #define FLASH_LAYER 18
 #define FULLSCREEN_LAYER 18.1
 #define UI_DAMAGE_LAYER 18.2
 
-#define HUD_PLANE 19
+#define HUD_PLANE 32680
```

Comment:
why are these numbers so high? Not that it matters, but seems rather arbitrary

Reply:
It's close to 32767 which is maximum plane value. No z-level planes should be higher than hud, so this is good way to ensure it